File: scala/dataSetTest.scala

object Test{
	def main(args: Array[String]) = {
		test()
	}
	def test() = {
		val a = Set("apple", "banana", "orange", "apple")
		for(k <- a){
			println(k)
		}
	}

	def test():Unit = {
		val a = Set("apple", "banana")
		a.foreach{
			k => println(k)
		}
	}
}

File: kotlin/dataSetTest.kt

fun test(){
	val a = setOf("apple", "banana")
	for(k in a){
		println(k)
	}
}

fun test(){
	var a = setOf("apple", "banana")
	a = a.plus("orange")
	for(k in a){
		println(k)
	}
}